packages/mds-api-server/middleware/http-context.ts

```diff
@@ -0,0 +1,10 @@
+import httpContext from 'express-http-context'
+import express from 'express'
+import { ApiRequest, ApiResponse } from '../@types'
+
+export const HttpContextMiddleware = () => httpContext.middleware
+
+export const RequestIdMiddleware = () => (req: ApiRequest, res: ApiResponse, next: express.NextFunction) => {
+  httpContext.set('x-request-id', res.get('x-request-id'))
+  return next()
+}
```

Comment:
Probably ought to just merge these into one middleware, would avoid us having to double-init at the top of the file